ddcz/forms/authentication.py

```diff
@@ -38,3 +40,197 @@ def get_users(self, email):
+        "#",
+        "$",
+        "^",
+        "&",
+        "*",
+        "{",
+        "}",
+        "[",
+        "]",
+        "(",
+        ")",
+        ":",
+        ";",
+        "=",
+        "+",
+        "?",
+        "|",
+        ",",
+        "~",
+    )
+
+    nick_uzivatele = forms.CharField(
+        label="",
+        widget=forms.TextInput(
+            attrs={"placeholder": "Zadej svůj nick", "id": "nickname"}
+        ),
+    )
+
+    email = forms.EmailField(
+        label="",
+        widget=forms.EmailInput(
+            attrs={"placeholder": "Zadej emailovou adresu", "id": "email"}
+        ),
+    )
+
+    jmeno = forms.CharField(
+        label="",
+        widget=forms.TextInput(
+            attrs={"placeholder": "Křestní jméno", "id": "first_name"}
+        ),
+    )
+
+    prijmeni = forms.CharField(
+        label="",
+        widget=forms.TextInput(attrs={"placeholder": "Příjmení", "id": "last_name"}),
+    )
+
+    osloveni = forms.CharField(
+        label="",
+        widget=forms.TextInput(attrs={"placeholder": "Oslovení", "id": "addressing"}),
+    )
+
+    pohlavi = forms.ChoiceField(
+        label="",
+        widget=forms.Select(attrs={"id": "sex"}),
+        choices=SEX_CHOICES.items(),
+    )
+
+    vek = forms.IntegerField(
+        label="",
+        widget=forms.NumberInput(attrs={"placeholder": "Tvůj věk", "id": "age"}),
+    )
+
+    duvod_registrace = forms.CharField(
+        required=False,
+        label="",
+        widget=forms.Textarea(
+            attrs={
+                "placeholder": "Co tě dovedlo k rozhodnutí zaregistrovat se na náš web? Je nějaký speciální důvod, proč bys rád patřil mezi nás?",
+                "id": "motive",
+            }
+        ),
+    )
+
+    kamaradi_na_webu = forms.CharField(
+        required=False,
+        label="",
+        widget=forms.Textarea(
+            attrs={
+                "placeholder": "Znáš ve městě někoho mimo Virga?",
+                "id": "web_friends",
+                "class": "form-control",
+            }
+        ),
+    )
+
+    odkud_znas_web = forms.CharField(
+        required=False,
+        label="",
+        widget=forms.Textarea(
+            attrs={
+                "placeholder": "Odkud ses dozvěděl o existenci tohoto města (webu)?",
+                "id": "source",
+            }
+        ),
+    )
+
+    gdpr = forms.ChoiceField(
+        label="",
+        widget=forms.Select(attrs={"id": "gdpr"}),
+        choices=GDPR_CHOICES.items(),
+    )
+
+    class Meta:
+        model = UzivateleCekajici
+        fields = [
+            "nick_uzivatele",
+            "email",
+            "jmeno",
+            "prijmeni",
+            "pohlavi",
+            "osloveni",
+            "datum",
+            "patron",
+            "primluvy",
+            "popis_text",
+        ]
+
+    # Setting the data
+    @classmethod
+    def set_for_save(cls, data):
+        data["datum"] = int(datetime.now().timestamp())
```

Comment:
FWIW, it's better to use `int(time)` from [time](https://docs.python.org/3/library/time.html#time.time)